lib/assessment.js

```diff
@@ -39,7 +61,11 @@ var newArray2 = function newArray2(length, defaultsFunction) {
 // The function you write should return
 // the object passed in as an argument or the object created
 var addMethod = function addMethod(propertyName, method, obj) {
-
+  if(!obj){
+    obj = {};
+  }
+  obj[propertyName] = method;
+  return obj;
```

Comment:
VERY nice! Spot on with this one.

Another way that you could have written the first bit is `obj = obj || {};`.
Can you see why?